examples/tts/conf/rad-tts_dec.yaml

```diff
@@ -0,0 +1,277 @@
+        stresses: True
+        chars: True
+        space: ' '
+        silence: null
+        apostrophe: True
+        sep: '|'
+        add_blank_at: null
+        pad_with_space: True
+        g2p:
+          _target_: "nemo.collections.tts.torch.g2ps.EnglishG2p"
+          phoneme_dict: ${phoneme_dict_path}
+          heteronyms: ${heteronyms_path}
+          phoneme_probability: 0.5
+          #adlr_symbol_id_mapper: ${adlr_symbol_id_mapper_path}
+    dataloader_params:
+      drop_last: false
+      shuffle: true
+      batch_size: 8
+      num_workers: 8
+      pin_memory: false
+
+  validation_ds:
+    dataset:
+      _target_: "nemo.collections.tts.torch.data.TTSDataset"
+      manifest_filepath: ${validation_datasets}
+      sample_rate: ${sample_rate}
+      sup_data_path: ${sup_data_path}
+      sup_data_types: ${sup_data_types}
+      n_fft: ${n_fft}
+      win_length: ${n_window_size}
+      hop_length: ${n_window_stride}
+      window: ${window}
+      n_mels: ${n_mels}
+      lowfreq: ${lowfreq}
+      highfreq: ${highfreq}
+      max_duration: null
+      min_duration: 0.1
+      ignore_file: null
+      trim: False
+      pitch_fmin: ${pitch_fmin}
+      pitch_fmax: ${pitch_fmax}
+      
+      
+        
+      text_tokenizer:
+        _target_: "nemo.collections.tts.torch.tts_tokenizers.EnglishPhonemesTokenizer"
+        punct: True
+        stresses: True
+        chars: True
+        space: ' '
+        silence: null
+        apostrophe: True
+        sep: '|'
+        add_blank_at: null
+        pad_with_space: True
+        g2p:
+          _target_: "nemo.collections.tts.torch.g2ps.EnglishG2p"
+          phoneme_dict: ${phoneme_dict_path}
+          heteronyms: ${heteronyms_path}
+          phoneme_probability: 0.5
+          #adlr_symbol_id_mapper: ${adlr_symbol_id_mapper_path}
+
+    dataloader_params:
+      drop_last: false
+      shuffle: false
+      batch_size: 8
+      num_workers: 8
+      pin_memory: false
+
+  optim:
+    name: RAdam
+    lr: 0.0001
+    betas: [0.9, 0.98]
+    weight_decay: 0.000001
+
+    sched:
+      name: exp_decay
+      warmup_steps: 40000
+      last_epoch: -1 
+      d_model: 1  # Disable scaling based on model dim 
+  trainerConfig:
+    sigma: 1
+    iters_per_checkpoint: 3000
+    seed: null
+    ignore_layers: []
+    finetune_layers: []
+    include_layers: []
+    with_tensorboard: true
+    dur_loss_weight: 1
+    ctc_loss_weight: 1
+    mask_unvoiced_f0: false
+    log_step: 1
+    binarization_start_iter: 6000
+    kl_loss_start_iter: 18000
+    loss_weights:
+        ctc_loss_weight: 0.1
+        dur_loss_weight: 1.0
+        f0_loss_weight: 1.0
+        energy_loss_weight: 1.0
+        vpred_loss_weight: 1.0
+    unfreeze_modules: "all"
+        
+
+    
+  load_from_checkpoint: False
+  init_from_ptl_ckpt: ${ckpt_path}
+  modelConfig:
+        _target_: "nemo.collections.tts.modules.radtts.RadTTSModule"
+        n_speakers: 1
+        n_speaker_dim: 16
```

Comment:
It seems `n_speaker_dim` is the dimension of speaker embeddings. Could you rename it as `speaker_embedding_dim`, just like `symbols_embedding_dim`?

Reply:
are we already using speaker_embedding_dim in NeMo? If not I want to be consistent with ADLR for now. That's why I kept it as n_speaker_dim.